openbazaard.go

```diff
@@ -481,6 +481,9 @@ func (x *Start) Execute(args []string) error {
 		return err
 	}
 	onionAddrString := "/onion/" + onionAddr + ":4003"
+	// add websocket support for browser peers
+	cfg.Addresses.Swarm = append(cfg.Addresses.Swarm, "/ip4/0.0.0.0/tcp/9005/ws")
+	cfg.Addresses.Swarm = append(cfg.Addresses.Swarm, "/ip6/::/tcp/9005/ws")
```

Comment:
@jackkleeman can you remove these two lines and add these addresses to the config in repo/config/InitConfig